src/host/consoleInformation.cpp

```diff
@@ -337,9 +337,9 @@ const std::wstring& CONSOLE_INFORMATION::GetTitle() const noexcept
 // - <none>
 // Return Value:
 // - a new wstring containing the combined prefix and title.
-const std::wstring CONSOLE_INFORMATION::GetTitleAndPrefix() const
+const std::wstring& CONSOLE_INFORMATION::GetTitleAndPrefix() const

```

Comment:
prefer a view. Terminal's impl can't return a wstring reference. Views are _always_ safer.

Reply:
(I was trying to build Terminal with this PR in, and ran into it :smile:)

Reply:
it's been viewified.